cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySqlClient.java

```diff
@@ -0,0 +1,159 @@
+                                                     ResultSet.HOLD_CURSORS_OVER_COMMIT);
+      ResultSet rsSource = statement1.executeQuery(getSourceQuery);
+      ResultSet rsTarget = statement2.executeQuery(getTargetQuery);
+      return compareResultSetData(rsSource, rsTarget);
+    }
+  }
+
+  /**
+   * Compares the result Set data in source table and sink table..
+   *
+   * @param rsSource result set of the source table data
+   * @param rsTarget result set of the target table data
+   * @return true if rsSource matches rsTarget
+   */
+  public static boolean compareResultSetData(ResultSet rsSource, ResultSet rsTarget) throws SQLException {
+    ResultSetMetaData mdSource = rsSource.getMetaData();
+    ResultSetMetaData mdTarget = rsTarget.getMetaData();
+    int columnCountSource = mdSource.getColumnCount();
+    int columnCountTarget = mdTarget.getColumnCount();
+    Assert.assertEquals("Number of columns in source and target are not equal", columnCountSource, columnCountTarget);
+    while (rsSource.next() && rsTarget.next()) {
+      int currentColumnCount = 1;
+      while (currentColumnCount <= columnCountSource) {
+        String columnTypeName = mdSource.getColumnTypeName(currentColumnCount);
+        int columnType = mdSource.getColumnType(currentColumnCount);
+        String columnName = mdSource.getColumnName(currentColumnCount);
+        if (columnType == Types.TIMESTAMP) {
+          GregorianCalendar gc = new GregorianCalendar(TimeZone.getTimeZone("UTC"));
+          gc.setGregorianChange(new Date(Long.MIN_VALUE));
+          Timestamp sourceTS = rsSource.getTimestamp(currentColumnCount, gc);
+          Timestamp targetTS = rsTarget.getTimestamp(currentColumnCount, gc);
+          Assert.assertTrue(String.format("Different values found for column : %s", columnName),
+                            sourceTS.equals(targetTS));
+        } else {
+          String sourceString = rsSource.getString(currentColumnCount);
+          String targetString = rsTarget.getString(currentColumnCount);
+          Assert.assertTrue(String.format("Different values found for column : %s", columnName),
+                            String.valueOf(sourceString).equals(String.valueOf(targetString)));
+        }
+        currentColumnCount++;
+      }
+    }
+    Assert.assertFalse("Number of rows in Source table is greater than the number of rows in Target table",
+                       rsSource.next());
+    Assert.assertFalse("Number of rows in Target table is greater than the number of rows in Source table",
+                       rsTarget.next());
+    return true;
+  }
+
+  public static void createSourceTable(String sourceTable) throws SQLException, ClassNotFoundException {
+    try (Connection connect = getCloudSqlConnection(); Statement statement = connect.createStatement()) {
+      String createSourceTableQuery = "CREATE TABLE IF NOT EXISTS " + sourceTable + "(id int, lastName varchar(255), PRIMARY KEY (id))";
```

Comment:
Same here, I know these things should be flagged  in review but (@priyabhatnagar25  & @IshaCS ) you both have been working on this and didn't see this is wrong it shouldn't be hardcoded.